src/Auth0Client.ts

```diff
@@ -1147,22 +1123,10 @@ export class Auth0Client {
         throw new Error('Invalid state');
       }
 
-      const {
-        scope,
-        audience,
-        redirect_uri,
-        cacheMode,
-        timeoutInSeconds,
-        detailedResponse,
-        ...customOptions
-      } = options;
-
       const tokenResult = await oauthToken(
         {
           ...this.customOptions,
-          ...customOptions,
-          scope,
-          audience,
+          ...options.authorizationParams,
```

Comment:
I think that previously, `this.customOptions` used to contain things like `screen_hint` when set while creating an instance of our SDK. Which would mean we would need to include `...this.options.authorizationParams`.

I would even think we should be able to drop `...this.customOptions`, as that's replaced by `this.options.authorizationParams` once we drop `this.customOptions`. I know we talked about dropping `this.customOptions` in a follow up PR, but it looks like we are changing behavior here while I think we should not, so it might make sense to look into that in this PR, what do you think?

Reply:
Yeah that makes sense, I'll look to remove customOptions in this PR

Reply:
customOptions is no longer removed, had to add a `...this.options.authorizationParams` to this call to pull in default custom parameters set on initialization